code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -120,7 +120,40 @@
 	w_class = WEIGHT_CLASS_HUGE
 	hitsound = 'sound/weapons/sear.ogg'
 	damtype = BURN
-	attack_verb = list("punched", "cross countered", "pummeled")
+	light_color = "#E09D37"
+	attack_verb = list("singed", "seared", "burnt", "scorched")
+	var/lit = TRUE
+
+/obj/item/nullrod/godhand/Initialize()
+	. = ..()
+	update_brightness()
+
+/obj/item/nullrod/godhand/proc/update_brightness()
+	set_light(2)
+
+/obj/item/nullrod/godhand/get_heat()
+	return 1500
+
+/obj/item/nullrod/godhand/attack(mob/living/carbon/M, mob/user)
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		to_chat(user, "<span class='warning'>You don't want to harm other living beings!</span>")
+	if(!isliving(M) || !ismob(M))
+		return
+	M.IgniteMob()
```

Comment:
So it'll ignite someone even if you only want to light their cig?

Reply:
Same code is present with normal lighters, so keeping it for consistency

Reply:
That might be subject to change in the future aswell, no need for consistency if it is weird.
If you think that the godhand should ignite people when you light a cig for them, keep it in, if not, change it